specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/SAPVirtualInstance.json

```diff
@@ -2446,6 +2449,11 @@
         "appResourceGroup": {
           "type": "string",
           "description": "The application resource group where SAP system resources will be deployed."
+        },
+        "isRestrictedUse": {
+          "description": "A boolean, if true, specifies that the user is aware about the given infrastructure configuration to be restricted.",
```

Comment:
let's get this validated from PMs, as it'll be visible in Docs exactly as we add it here.

Reply:
Sure

Reply:
Good with the parameter usage